content/docs/gto/get-started.md

```diff
@@ -0,0 +1,170 @@
+│ segment  │ v0.4.1   │ v0.4.1 │ -       │ -          │
+│ cv-class │ v0.1.14  │ -      │ -       │ -          │
+╘══════════╧══════════╧════════╧═════════╧════════════╛
+```
+
+## Assigning stages
+
+Let's assume the version we just registered looks very promising, and we want to
+promote it to `dev` to test it:
+
+```cli
+$ gto assign cv-class --stage dev
+Created git tag 'cv-class#dev#1' that assigns stage to version 'v0.1.14'
+To push the changes upstream, run:
+    git push origin cv-class#dev#1
+
+# stage assignment can be seen with `gto show`:
+$ gto show
+╒══════════╤══════════╤═════════╤═════════╤════════════╕
+│ name     │ latest   │ #dev    │ #prod   │ #staging   │
+╞══════════╪══════════╪═════════╪═════════╪════════════╡
+│ churn    │ v3.1.1   │ v3.1.1  │ v3.0.0  │ v3.1.0     │
+│ segment  │ v0.4.1   │ v0.4.1  │ -       │ -          │
+│ cv-class │ v0.1.14  │ v0.1.14 │ -       │ -          │
+╘══════════╧══════════╧═════════╧═════════╧════════════╛
+```
+
+We can push to Git repository to start a CI job!
+
+## Act on registrations and assignments in CI/CD
+
+GTO [uses Git tags] to register artifact versions and assign them to stages.
+This means we can act upon these operations in any Git-based system such as many
+CI/CD platforms.
+
+[uses git tags]: /doc/gto/user-guide#git-tags-format
+
+<details>
+
+### Click to set up a working repository to try this.
+
+Let's fork the [example-gto repo](https://github.com/iterative/example-gto) repo
+(you'll need a [GitHub](https://github.com/signup) account first). For CI/CD to
+start, you'll need to enable it on the "Actions" page of your fork.
+
+1. [Fork the repo](https://github.com/iterative/example-gto/fork). Make sure you
+   uncheck "Copy the `main` branch only" to copy Git tags as well:
+   <img width="877" alt="image" src="https://user-images.githubusercontent.com/6797716/199275275-439335f4-6f54-4cd7-910d-fc29ad3c095c.png">
+
+2. Enable workflows in your repo for a new Git tag to trigger CI:
+   <img width="869" alt="image" src="https://user-images.githubusercontent.com/6797716/199272682-dfd628bf-9599-4e85-a623-bf4a10c3d7e1.png">
+
+</details>
+
+Now let's push the Git tags we created locally to your repository (don't forget
+to replace `aguschin` with your username):
+
+```cli
+$ git remote update origin https://github.com/aguschin/example-gto
+$ git push origin cv-class@v0.1.14 cv-class#dev#1
+ * [new tag]         cv-class@v0.1.14 -> cv-class@v0.1.14
+ * [new tag]         cv-class#dev#1 -> cv-class#dev#1
+```
+
+<details>
+
+### Or just repeat the assignment for your new repo
+
+We can do the same thing we did locally, but for your remote repo (don't forget
+to replace `aguschin` with your username):
+
+```cli
+# since we didn't register a version on remote, GTO will do that for us:
+$ gto assign cv-class --stage dev \
+    --repo https://github.com/aguschin/example-gto
+Created git tag 'cv-class@v0.1.14' that registers a version
```

Comment:
Why would `gto assign` create the `gto register` tag here? Is it done even if it already exists (which would be the case here)?

I'm not sure this output matches what the user would see when following the GS. See my concurrent suggestion where I edit it, but I didn't have time to actually try it (someone should double check).

Reply:
By default, assign creates a version, that's why the tag for version is created. In your case, it won't be created since it exists already.

Reply:
Good to know 😬